test/zipkin-simple.test.js

```diff
@@ -234,26 +230,26 @@ describe("Zipkin client", function () {
 					done()
 				})
 
-				Client.clientRecv(traceData, {
+				Client.sendClientRecv(traceData, {
```

Comment:
For the name, see the comment above.

For the service and name explicitly here, why? Do you think I should store them in the returned traceData and reuse them?